content/themes/casper

```diff
@@ -1 +1 @@
-Subproject commit de91a4006b49868aa5ffc1f7af3b79abe1595897
+Subproject commit 1c092c7be098bed496eaa2f194a3ae39dd23659d
```

Comment:
Not sure why this keeps creeping back in on different PR's, but this needs to be removed otherwise the Casper version will be reverted.